aimmo-game-creator/worker_manager.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, *args, **kwargs):
         super(KubernetesWorkerManager, self).__init__(*args, **kwargs)
 
     def _create_game_rc(self, id, environment_variables):
-        environment_variables['SOCKETIO_RESOURCE'] = "game/%s/socket.io" % id
```

Comment:
We only have one socket.io resource exposed. The problem came from the fact that we could not set the resource to game/%s/socket.io in Unity, so we changed the backend resource to be /socket.io